cddagl/ui.py

```diff
@@ -497,7 +589,7 @@ def __init__(self):
 
                 try:
                     self.pipe = SimpleNamedPipe('cddagl_instance')
-                except (OSError, PyWinError):
+                except (OSError):

```

Comment:
PyWinError should be generalized to some other which is platform independent. I excluded it for now.